QModManager/Patcher.cs

```diff
@@ -73,7 +73,7 @@ internal static void StartLoadingMods()
                     Console.WriteLine("There was an error creating the QMods directory");
                     Console.WriteLine("Please make sure that you ran Subnautica from Steam");
                 }
-                try
+                else try
```

Comment:
`else try` isn't a very common structure to see.  
For readability, and less confusion, I would suggest either nesting the `try-catch` inside the `else` block or using early exit return statements for reduced nesting.

As a general rule, always leave an in-line comment whenever you do an early exit out of a long method or code block, just to make sure everyone understands what's going on.

Reply:
Addressed in commit 35a9bad